server/text_generation_server/models/custom_modeling/flash_cohere_modeling.py

```diff
@@ -297,8 +298,8 @@ def forward(
             # flash attention
             attn_output = attention(
                 query,
-                kv_cache[0] if SYSTEM != "ipex" else key,
-                kv_cache[1] if SYSTEM != "ipex" else value,
+                kv_cache[0] if PAGED_KV else key,
```

Comment:
I was making the same change in #2575. Maybe we can settle on the same name to avoid merge conflicts? I was using `PREFILL_IN_KV_CACHE` to be as descriptive as possible.

Reply:
Yes, added similar to your PR